Starting style guide with one-param-per line rule

mtlynch opened the pull request:
This starts off the Ansible style guide with framing test and adds the first
Ansible style rule.

The first style rule enforces the one parameter per line rule, as discussed
in #1.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/m-lab/ansible-style-guide/8)
<!-- Reviewable:end -->

nkinkade commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

_[README.md, line 39 [r1]](https://reviewable.io:443/reviews/m-lab/ansible-style-guide/8#-KI2wMX3s-zkz2UPdDIt:-KI2wMX3s-zkz2UPdDIu:-978779264) ([raw file](https://github.com/m-lab/ansible-style-guide/blob/2d936e8e3fa87498c38c090a831ce3ffd865a3b1/README.md#L39)):_

> ``` Markdown
> #### Decision
> 
> Use line breaks to separate parameters to Ansible modules, for example:
> ```

I might make this "Use line breaks and indention..." just to be 100% clear.

---

_[README.md, line 47 [r1]](https://reviewable.io:443/reviews/m-lab/ansible-style-guide/8#-KI2wc4Kv9fhGhhaYm8k:-KI2wc4Kv9fhGhhaYm8l:-2103377100) ([raw file](https://github.com/m-lab/ansible-style-guide/blob/2d936e8e3fa87498c38c090a831ce3ffd865a3b1/README.md#L47)):_

> ``` Markdown
>         owner=bar
>         group=fooers
> ```
> 
> ```
> ```

Would it be worth the while to demonstrate both the good and the bad, like Google does in its style guides?

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/ansible-style-guide/8)_

<!-- Sent from Reviewable.io -->

mtlynch commented:
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

_[README.md, line 39 [r1]](https://reviewable.io:443/reviews/m-lab/ansible-style-guide/8#-KI2wMX3s-zkz2UPdDIt:-KI4RavDH9EyeIsEzYhD:1125174564) ([raw file](https://github.com/m-lab/ansible-style-guide/blob/2d936e8e3fa87498c38c090a831ce3ffd865a3b1/README.md#L39)):_
<details><summary>

<i>Previously, nkinkade wrote…</i></summary>



> I might make this "Use line breaks and indention..." just to be 100% clear.
> </details>
> Good suggestion, added.

---

_[README.md, line 47 [r1]](https://reviewable.io:443/reviews/m-lab/ansible-style-guide/8#-KI2wc4Kv9fhGhhaYm8k:-KI4R_THqBMYNyxmvj9o:2111259327) ([raw file](https://github.com/m-lab/ansible-style-guide/blob/2d936e8e3fa87498c38c090a831ce3ffd865a3b1/README.md#L47)):_
<details><summary>

<i>Previously, nkinkade wrote…</i></summary>



> Would it be worth the while to demonstrate both the good and the bad, like Google does in its style guides?
> </details>
> That makes sense. Added.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/ansible-style-guide/8)_

<!-- Sent from Reviewable.io -->
